README.md

```diff
@@ -5,11 +5,10 @@
 # 📗 Table of Contents
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request professional 👍